cmd/flags.go

```diff
@@ -59,6 +59,11 @@ var globalFlags = []cli.Flag{
 		Usage:  "enable debug output",
 		EnvVar: envPrefix + "DEBUG",
 	},
+	cli.StringSliceFlag{
+		Name:   "resolve",
+		Usage:  "resolves HOST[:PORT] to an IP address",
```

Comment:
```suggestion
		Usage:  "resolves HOST[:PORT] to an IP address. Example: minio.local:9000=10.10.75.1",
```

(if you do go for `=`)